.gitignore

```diff
@@ -1,3 +0,0 @@
-_site/
```

Comment:
Should not remove this file.  Rather - it should simply not be included in the pull request.